data/random-teams.js

```diff
@@ -1835,9 +1835,16 @@ class RandomTeams extends Dex.ModdedDex {
 				pokemon.push(set);
 
 				if (pokemon.length === 6) {
-					// Set Zoroark's level to be the same as the last Pokemon
 					let illusion = teamDetails['illusion'];
-					if (illusion) pokemon[illusion - 1].level = pokemon[5].level;
+					if (illusion) {
+						// Swap Zoroark's slot if it is the last Pokemon
+						if (illusion === 5) {
```

Comment:
I'm confused, what's going on here? Can't Zoroark be in any slot?

Reply:
*if* it happens to be in the last slot, swap it with the second-to-last slot so that Illusion actually *does* something

Reply:
Ohhhh. Maybe that comment could be rewritten to be clearer.